Source/multi.cpp

```diff
@@ -32,7 +32,7 @@ bool gbSomebodyWonGameKludge;
 TBuffer sgHiPriBuf;
 char szPlayerDescript[128];
 uint16_t sgwPackPlrOffsetTbl[MAX_PLRS];
-PkPlayerStruct netplr[MAX_PLRS];
+PlayerPack netplr[MAX_PLRS];

```

Comment:
How about `CompactPlayer`? Pack seams a bit ambiguous. Could also be `PackedPlayer` (swapping the words is also fine).

Reply:
I searched a bit about this before renaming and "Pack" seems fine in this context:
https://wikidiff.com/pack/package

`CompactPlayer` would be weird to me as it would indicate a "variation of player", as if it was some sort of inheritance. `PackedPlayer` would fall on this category a bit as well IMHO.

If you feel strongly about it, I can rename to whatever you want though.

Reply:
thinking about it this is acutally used as the payload in some network packages so might just be ok as is